elasticsearch-scylla/README.md

```diff
@@ -0,0 +1,95 @@
+# Mine data with Elasticsearch and ScyllaDB
+<p align=center>
+
+The purpose of this demo is to show how to feed data from Twitter into Elasticsearch and Scylla for data analytics purposes. The Twitter app will search for a specific Twitter hashtag and feed data into Scylla. To feed the data from Scylla to Elasticsearch, simply issue a curl command referenced in this README to begin the data dump.
+
+### Architecture
+
+##### Scylla Architecture
+Three Scylla 1.7 nodes
+
+##### Elasticsearch Architecture
+Two Elasticsearch nodes (Master, data).
+![Pic](https://raw.githubusercontent.com/scylla/scylla-code-samples/master/elasticsearch-scylla/diagram.png)
+
+### Prerequisites
+
+1. Docker for [Mac](https://download.docker.com/mac/stable/Docker.dmg) or [Windows](https://download.docker.com/win/stable/InstallDocker.msi).
+2. This Git repository
+3. 3GB of RAM or greater for Docker
+4. (Optional) [Twitter API credentials](https://dev.twitter.com/)
+
+### Building the images
+```
+unzip elasticsearch-scylla-master.zip
+cd elasticsearch-scylla
+docker-compose build
+```
+
+To use the Twitter app to mine data from Twitter, modify the twitter section of docker-compose.yml with your developer API credentials and desired Twitter topic.
```

Comment:
how do I do that?
Is it on this page: https://apps.twitter.com/app/new
what values should I use?